llvm/lib/Transforms/IPO/SampleProfileMatcher.cpp

```diff
@@ -127,8 +147,37 @@ void SampleProfileMatcher::findProfileAnchors(const FunctionSamples &FS,
   }
 }
 
-LocToLocMap SampleProfileMatcher::longestCommonSequence(
-    const AnchorList &AnchorList1, const AnchorList &AnchorList2) const {
+bool SampleProfileMatcher::isFunctionEqual(const FunctionId &IRFuncName,
```

Comment:
this is not a good name. should this still be called `functionMatchesProfile`?

Reply:
changed to `functionMatchesProfile`